winFixDeprecation.ps1

```diff
@@ -0,0 +1,6 @@
+$InputFiles = Get-Item "lib/proto/*.js"
+$OldString  = 'new Buffer'
+$NewString  = 'Buffer.from'
+$InputFiles | ForEach {
+    (Get-Content -Path $_.FullName).Replace($OldString,$NewString) | Set-Content -Path $_.FullName
+}
```

Comment:
Missing `EOF`.

Reply:
I was thinking of that too, but I just can't find out how to get the EOF character in PowerShell.